### ISSUE TITLE
STK 0.9 Mac OS X Version: 10.9 necessary?

### ISSUE THREAD

**Issue by CreativeForge** [supertuxkart/stk-code#2063]
The new STK 0.9rc1 doesn't run on "older" Macs. Maybe it was just built for 10.9 accidentally.
Is the OS minimum really 10.9. There are still Macs with good Hardware which runs Snow Leopard (10.6).

**Comment by hiker** [comment]
Here what @auriamg wrote in the forum (with some fixes to the version numbers):

"Unfortunately, Apple makes it very difficult to support older OSX versions, since they are not available for download, and systems before 10.9 are unsupported by apple"

It would be great if you have access to to an older system and could compile and bundle STK (with some help from us if necessary of course)!

**Comment by CreativeForge** [comment]
"Unfortunately, Apple makes it very difficult to support older OSX versions, since they are not available for download, and systems before 10.9 are unsupported by apple"
- I totally agree with Auria! I had already problems with backward compatibility on Macs earlier. But I thought I'd give it a try....

I spent the last days trying to build STK on older systems (10.6 & 10.8). Here's a little summary:
- OS X 10.6:
OpenGL 3 is not fully supported and is one of the (many) problems here...
- OS X 10.8:
On OS X 10.8 and previous, clang uses an older gcc-4.2 std library which doesn't have support for many functions like std::forward (for example used in: https://github.com/supertuxkart/stk-code/blob/ef89e168f8b665e5fc089b1a3b333d3dfa948dbf/src/utils/string_utils.hpp). So I tried to find a workaround with macport and newer gcc libraries but couldn't get any promising results. Additional there are some more errors I get from cmake on 10.8, like missing OpenMP sources. I wasn't able to resolve everything and also don't have time the next days. So, I personally am not able to build a proper 10.8 version by myself.

So, I think the easiest way is to have a newer OS ;)
